packages/api/src/webhooks/cannon.test.ts

```diff
@@ -327,21 +385,69 @@ describe("webhook cannon", () => {
         streamId: "streamid",
         event: "stream.started",
         userId: nonAdminUser.id,
-        projectId: nonAdminUser.defaultProjectId,
+        projectId: nonAdminProject.id,
       });
 
       await Promise.all(sems.map((s) => s.wait(3000)));
       expect(calledFlags).toEqual([true, true]);
     });
 
-    it("should send multiple events to same webhook", async () => {
```

Comment:
Could you keep these tests separate? It gets hard to understand/debug if the test case is too long and from the description it seems like they are testing different things.